version.js

```diff
@@ -36,7 +36,9 @@ if (program.debug) {
 	console.log(`use stack version ${stackVersion}`);
 }
 
-const REACT_VERSION = '^16.0.0';
+const REACT_VERSION = process.env.REACT_VERSION || '^16.0.0';
+console.log('REACT_VERSION: ', REACT_VERSION);
```

Comment:
should be removed i guess

Reply:
No please keep it for debug in the CI
Because we need to handle both and it comes from ENV, it nice to have it logged here